### ISSUE TITLE
[FEATURE] Add Zenodo DOI badge to README

### ISSUE THREAD

**Issue by sandeep-ps** [rokwire/illinois-app#1577]
**Describe the solution you'd like**
We recently created a DOI entry in Zenodo for citation purposes. We need to add the DOI badge to the README.

```[![DOI](https://zenodo.org/badge/279491766.svg)](https://zenodo.org/badge/latestdoi/279491766)```

**Comment by vburgett** [comment]
Can we close this ticket, @sandeep-ps? I think so but thought I would check with you first. Thank you!

**Comment by sandeep-ps** [comment]
Yes, this can be closed. If useful, someone can add ```[![DOI](https://zenodo.org/badge/279491766.svg)](https://zenodo.org/badge/latestdoi/279491766)``` to the top of the README, and which will let others know how to cite this repository in any publication.

The Zenodo entry for the Illinois app, and corresponding citation details exits at https://zenodo.org/badge/latestdoi/279491766 and gets automatically updated during each release. If the team prefers a static customized author list (e.g., just an organization name), a CITATION.CFF file can be added to the repository. Details are here: https://citation-file-format.github.io/
